README.md

```diff
@@ -184,7 +184,7 @@ These GitHub repositories provide supplementary resources for Rush Stack:
 | [/build-tests/rush-lib-declaration-paths-test](./build-tests/rush-lib-declaration-paths-test/) | This project ensures all of the paths in rush-lib/lib/... have imports that resolve correctly. If this project builds, all `lib/**/*.d.ts` files in the `@microsoft/rush-lib` package are valid. |
 | [/build-tests/rush-project-change-analyzer-test](./build-tests/rush-project-change-analyzer-test/) | This is an example project that uses rush-lib's ProjectChangeAnalyzer to  |
 | [/build-tests/rush-redis-cobuild-plugin-integration-test](./build-tests/rush-redis-cobuild-plugin-integration-test/) | Tests connecting to an redis server |
-| [/build-tests/set-webpack-public-path-plugin-test](./build-tests/set-webpack-public-path-plugin-test/) | Building this project tests the set-webpack-public-path-plugin |
+| [/build-tests/set-webpack-public-path-plugin-test](./build-tests/set-web pack-public-path-plugin-test/) | Building this project tests the set-webpack-public-path-plugin |
```

Comment:
Do you even check the changes you propose? Or is this based on some automatic tool?